styles/theme.ts

```diff
@@ -10,11 +10,21 @@ export const theme = createTheme({
       contrastText: '#000'
     },
     secondary: {
+      light: '#DAEDEA',
       main: '#78BDB2',
       contrastText: '#000'
     }
   },
   typography: {
     fontSize: 12
```

Comment:
Fonsize borde vara 16, för tillgänglightetnsskull, men också för att formulär annars zoomar in automatiskt.

Reply:
Fixar